### ISSUE TITLE
App crash when bluetooth headphones disconnect

### ISSUE THREAD

**Issue by 34fame** [hrkfdn/ncspot#504]
**Describe the bug**
I remove my headphones.  After a bit they turn off automatically.  When I come back the app has crashed.  I restart it after headphones reconnect and back to normal.  Crash results in attached screenshot.

Fun app, thanks!

**To Reproduce**
1. Listen with bluetooth headphones
2. Turn off headphones while listening
3. See error

**Expected behavior**
Stop/pause playback until connection resumes.

**System (please complete the following information):**
 - OS: macOS
 - Terminal: Terminal
 - Version: 0.6.0
 - Installed from: brew

**Backtrace/Debug log**
Sorry, not savvy enough for these instructions

![Screen Shot 2021-04-28 at 1 26 05 PM](https://user-images.githubusercontent.com/49797484/116454157-4df39f00-a825-11eb-9f08-b844de9acdb2.png)

**Comment by 34fame** [comment]
I was able to set env variable after all.  New screenshot attached.

![Screen Shot 2021-04-28 at 1 28 51 PM](https://user-images.githubusercontent.com/49797484/116454523-b17dcc80-a825-11eb-8acd-c8a2138f7e43.png)

**Comment by hrkfdn** [comment]
Does this help? https://github.com/hrkfdn/ncspot/issues/484#issuecomment-816930980

**Comment by 34fame** [comment]
Still results in error.

![Screen Shot 2021-05-04 at 9 33 20 AM](https://user-images.githubusercontent.com/49797484/117020129-c5a94a00-acbb-11eb-9c98-bd8052bdd65c.png)

**Comment by deed02392** [comment]
Unfortunately this also affects me with the HEAD install of `portaudio`:

||PaMacCore (AUHAL)|| Warning on line 521: err=''!obj'', msg=Unknown Error
||PaMacCore (AUHAL)|| Warning on line 441: err=''!obj'', msg=Unknown Error
||PaMacCore (AUHAL)|| Error on line 1332: err='-10851', msg=Audio Unit: Invalid Property Value
                                                       thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Internal PortAudio error', /Users/local/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/librespot-playback-0.2.0/src/audio_backend/portaudio.rs:124:46
                                                                                             note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by hrkfdn** [comment]
I don't think this can be fixed in ncspot. You could give the rodio backend a try and see if that improves things.
